lighthouse-core/report/html/renderer/performance-category-renderer.js

```diff
@@ -112,13 +112,20 @@ class PerformanceCategoryRenderer extends CategoryRenderer {
   /**
    * @param {LH.ReportResult.Category} category
    * @param {Object<string, LH.Result.ReportGroup>} groups
+   * @param {string=} environment
```

Comment:
maybe be specific about what's accepted...I assume `'PSI'=` works but that looks weird :)